crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -18,11 +18,11 @@
 
 module Api
   class Crowbar < Tableless
-    attr_reader :version
+    attr_reader :version, :addons
 
     def initialize
       @version = ENV["CROWBAR_VERSION"]
-      @addons = addons
+      @addons = features
```

Comment:
Why do this in `#initialize`?  If you're really worried about the calculation being repeated multiple times in a single request, then simply memoize the result the first time it's calculated, so that it's only calculated when it's really needed.